CHANGES.txt

```diff
@@ -1,6 +1,9 @@
 1.3.x (unreleased)
  * Removed use of Thrift describe_ring and replaced it with native Java Driver
    support for fetching TokenRanges (SPARKC-93)
+ * Decoupled PredicatePushDown logic from Spark (SPARKC-166)
```

Comment:
the content of `changes.txt` could autogenerated from Jira, with a bit of configuring Jira itself, and some scripting. I did that in the past in several project, works charming... Interest?